sinfonier-web/app/controllers/Topologies.java

```diff
@@ -203,8 +207,74 @@ public static void search(String status, String query, Date updated, int page) t
   public static void log(@Required String id) {
     try {
       Codes code200 = Codes.CODE_200;
+      Gson gson = new Gson();
       JsonObject data = new JsonObject();
-      data.addProperty("msg", client.getTopologyLog(id));
+      String logDataKey = id+"_logdata";
+      String startParam = request.params.get("start");
```

Comment:
I don't understand you, can you give me an example? start is a param, not part of the path. Can I use a binder for optional params too?